src/visning/Stillingsinfo.tsx

```diff
@@ -65,7 +66,11 @@ const Stillingsinfo: FunctionComponent<Props> = ({ stilling }) => {
                     )}
                 </Tabell>
                 {properties.applicationurl && (
-                    <Lenkeknapp href={properties.applicationurl}>Søk på stillingen</Lenkeknapp>
+                    <Lenkeknapp href={properties.applicationurl} onClick={() => {
+                        logEvent('stilling_søkeknapp', 'klikk')
```

Comment:
Vi er vel alltid på en stilling i denne appen
```suggestion
                        logEvent('søkeknapp', 'klikk')
```